src/new-client/src/App/Analytics/Analytics.tsx

```diff
@@ -1,34 +1,36 @@
-import { merge } from "rxjs"
-import { Subscribe } from "@react-rxjs/core"
 import { Loader } from "components/Loader"
-import { ProfitAndLoss, profitAndLoss$ } from "./ProfitAndLoss"
-import { Positions, positions$ } from "./Positions"
-import { PnL, pnL$ } from "./PnL"
-import {
-  AnalyticsStyle,
-  AnalyticsWrapper,
-  AnalyticsHeader,
-  AnalyticsInnerWrapper,
-} from "./styled"
+import { lazy, Suspense, useRef } from "react"
+import { analytics$ } from "services/analytics"
+import styled from "styled-components/macro"
+import { useHasItBeenVisible } from "utils/useHasItBeenVisible"
+const AnalyticsCore = lazy(() => import("./AnalyticsCore"))
 
-const analytics$ = merge(pnL$, profitAndLoss$, positions$)
+const AnalyticsWrapper = styled.div`
+  padding: 0.5rem 1rem 0.5rem 0;
+  user-select: none;
+  overflow: hidden;
 
+  @media (max-width: 750px) {
+    display: none;
+  }
+`
```

Comment:
> Is this lifted out for code-splitting? 

Nope, it's lifted out for tree-shaking :slightly_smiling_face:

This is the thing: CRA has serious problems with tree-shaking ATM. I will explain this better in tomorrow's demo, but the thing is that if we were not using CRA, then we would be able to put this inside the `styled` module and the resulting chunks would be the same that we got now... Well, they would be even better.

That being said, the main reason why I put it here, it's not because I want to tree-shake the rest of the "styled" components that we have in that file, as the difference in size would be minimal. The main reason why I put it here for a matter of consistency, and because I really think that whenever possible we should avoid lifting things for no good reason.. Meaning that this component is only used here, so I much rather to collocate it where it's being used, rather than having a hotchpotch of styled-components in an artificially lifted file. Also, this is consistent with most the other "wrappers" that we currently have, for instance [this one](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/6b535cb91b83a4b66428513648d96b2a86ecf50a/src/new-client/src/App/App.tsx#L9-L44), or [this one](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/6b535cb91b83a4b66428513648d96b2a86ecf50a/src/new-client/src/App/Trades/Trades.tsx#L9-L19), or [this one](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/master/src/new-client/src/App/LiveRates/LiveRates.tsx#L8-L26)...